src/pybind/mgr/snap_schedule/module.py

```diff
@@ -60,16 +63,20 @@ def handle_command(self, inbuf: str, cmd: Dict[str, str]) -> Tuple[int, str, str
     @CLIReadCommand('fs snap-schedule status')
     def snap_schedule_get(self,
                           path: str = '/',
-                          fs: Optional[str] = None,
+                          fs: str,
```

Comment:
non-default arg following default arg, would generate errors

Reply:
i guess let us provide default_fs to fs?

Reply:
Thanks for pointing out the **non-default arg after default args** issue
I missed restoring that part when implementing Patrick's suggestion.
I've restored the `fs` argument declaration as `fs: Optional[str] = None`